File: simulasi/ev.php
<?php
// Simulasi Expected Value (EV) judi online
// ⚠️ Edukasi, bukan untuk praktik judi

$probWin = 0.49;   // peluang menang
$probLose = 0.51;  // peluang kalah
$payoutWin = 1;    // untung jika menang
$payoutLose = -1;  // rugi jika kalah

$EV = ($probWin * $payoutWin) + ($probLose * $payoutLose);

echo "Expected Value per taruhan: $EV\n";
echo "Artinya: rata-rata rugi " . abs($EV * 100) . "% per taruhan\n";
?>

File: simulasi/ev.py
# simulasi/ev.py
# ⚠️ Edukasi, bukan untuk praktik judi
# Menghitung Expected Value (EV) sederhana

# Parameter permainan
prob_win = 0.49   # peluang menang
prob_lose = 0.51  # peluang kalah
payout_win = 1    # keuntungan jika menang
payout_lose = -1  # kerugian jika kalah

# Hitung EV
EV = (prob_win * payout_win) + (prob_lose * payout_lose)

print(f"Expected Value per taruhan: {EV:.2f}")
print(f"Artinya: rata-rata rugi {abs(EV*100):.1f}% per taruhan")
